src/App.jsx

```diff
@@ -0,0 +1,17 @@
+import "./CSS/App.css";
+import "./CSS/reusable.css";
+import "./CSS/Reviews.css";
+
+import Header from "./components/persistent/Header";
+import ReviewsPage from "./components/ReviewsPage";
+
+function App() {
+	return (
+		<div className="App">
+			<Header />
+			<ReviewsPage />
+		</div>
+	);
+}
+
+export default App;
```

Comment:
Don't forget to add your routers here - you're going to need them for subsequent tickets so I'd advise putting them in straight away.